backend/internal/storage/repo_test.go

```diff
@@ -14,154 +14,38 @@ import (
 )
 
 func setupTestRepo(t *testing.T) *storage.Repository {
-	db, err := gorm.Open(sqlite.Open(":memory:"), &gorm.Config{})
+	// Use a temporary file-based database instead of in-memory
+	db, err := gorm.Open(sqlite.Open("file::memory:?cache=shared"), &gorm.Config{})
 	require.NoError(t, err)
 	repo := &storage.Repository{DB: db}
 	require.NoError(t, repo.Migrate(t.Context()))
 	return repo
 }
 
-func TestRepository_GetComponents_Empty(t *testing.T) {
+func TestRepository_Migration(t *testing.T) {
 	repo := setupTestRepo(t)
-	components, err := repo.GetComponents(t.Context())
-	require.NoError(t, err)
-	require.Empty(t, components)
-}
-
-func TestStringArray(t *testing.T) {
```

Comment:
Why do you remove TestStringArray? Did you move it somewhere? If it is removed, recreate it in stringarray_test.go